examples/microsoft_devicecode_tenant_user.rs

```diff
@@ -0,0 +1,44 @@
+use oauth2::basic::BasicClient;
+use oauth2::devicecode::StandardDeviceAuthorizationResponse;
+use oauth2::reqwest::async_http_client;
+use oauth2::{AuthUrl, ClientId, DeviceAuthorizationUrl, Scope, TokenUrl};
+use std::error::Error;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let device_auth_url = DeviceAuthorizationUrl::new(
+        "https://login.microsoftonline.com/{tenant}/oauth2/v2.0/devicecode".to_string(),
+    )?;
+    let client = BasicClient::new(
+        ClientId::new("client_id".to_string()),
+        None,
+        AuthUrl::new(
+            "https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize".to_string(),
+        )?,
+        Some(TokenUrl::new(
+            "https://login.microsoftonline.com/{tenant}/v2.0/oauth2/token".to_string(),
```

Comment:
shouldn't this be `{tenant}/oauth2/v2.0/token`?

Reply:
I will change it. Thanks for finding it :)

Reply:
this is the same issue @randomairborne fixed in #220 for the common case

Reply:
@ramosbugs  ohh, I already changed that in this PR.

Reply:
@ramosbugs  ok you can merge it, I will just do a rebase

Reply:
@ramosbugs  rebase done